java/test/src/main/java/org/ray/api/test/MultiThreadingTest.java

```diff
@@ -55,6 +55,22 @@ public static String testMultiThreading() {
       Assert.assertEquals(arg, (int) obj.get());
     }, LOOP_COUNTER);
 
+    // Test creating multi actors
+    runTestCaseInMultipleThreads(() -> {
+      int arg = random.nextInt();
+      RayActor<Echo> echoActor1 = Ray.createActor(Echo::new);
+      try {
+        // Sleep a while so that another actor can be created before task is executed.
+        // To make sure test scenario about different actor tasks executed on different
+        // workers is covered.
+        Thread.sleep(10);
+      } catch (Exception e) {
+        e.printStackTrace();
+      }
```

Comment:
Nits:
1) Use `TimeUnit.MILLISECONDS.sleep`.
2) Don't catch `Exception`, catch the specific `InterruptedException`.
3) don't `e.printStackTrace();`, use `logger.warn`.